### PR TITLE
WEB: Add page about benchmarks

### PR DESCRIPTION
**Pull request by datapythonista**
CC: @DeaMariaLeon @rhshadrach @lithomas1 

This is not yet ready, the conbench url provided is not correctly rendering the css, and I need to add all the information about the OVH server, but opening early in case you have any feedback.

### CODE REVIEW

**Comment by lithomas1** [comment]
Hm, the ASV link for the OVH results seem to be dead add clicking on any of the times for the runs for the conbench results also gets me a 404.

Is this expected?

**Comment by DeaMariaLeon** [comment]
The links on the conbench runs don't work either. 
Plus if the menu is not available, one cannot give feedback.  

The server where I setup conbench is here and it works: http://57.128.112.95:5000/
Adding it to pandas' website caused this problem with css - I'm working on it. 
I think I found the reason, but to fix it I will have to take the server down for a bit.

**Comment by datapythonista** [comment]
Thanks for the heads up @lithomas1. I'm now synching the asv website to /benchmarks/asv/.  Hopefully we'll have the conbench site rendering with the css/js soon.

Any other feedback very welcome.

**Comment by datapythonista** [comment]
@lithomas1 @rhshadrach any feedback here before I merge? There will surely be follow ups on this, but I think for now it adds value as it is.

**Comment by DeaMariaLeon** [comment]
Just to be clear for other people who may read this, regarding:

>the conbench url provided is not correctly rendering the css

Actually, conbench does not currently support this kind of URL. It might in the future. Details about this are here: https://github.com/conbench/conbench/issues/1575#:~:text=We%20did%20not%20build%20(test%2C%20qualify)%20for%20deploying%20the%20Conbench%20web%20application%20at%20a%20non%2Droot%20path%20yet.

File: web/pandas/community/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## pandas benchmarks
+
+pandas benchmarks are implemented in the [asv_bench](https://github.com/pandas-dev/pandas/tree/main/asv_bench)
+directory of our repository. The benchmarks are implemented for the
+[airspeed velocity](https://asv.readthedocs.io/en/v0.6.1/) (asv for short) framework.
+
+The benchmarks can be run locally by any pandas developer. This can be done
+with the `asv run` command, and it can be useful to detect if local changes have
+an impact in performance, by running the benchmarks before and after the changes.
```
Comment (rhshadrach) [inline web/pandas/community/benchmarks.md:18]: Might link to here? https://pandas.pydata.org/docs/dev/development/contributing_codebase.html#id24

**Review by rhshadrach — APPROVED** [review]
lgtm - nice write up. One potential suggestion, but I'm good as-is.

**Comment by lithomas1** [comment]
This LGTM.

Thanks for writing this up.

File: web/pandas/community/benchmarks.md (hunk 1 above)
Comment (datapythonista) [reply web/pandas/community/benchmarks.md:18]: Very good point, added.

**Review by rhshadrach — APPROVED** [review]
lgtm

**Comment by rhshadrach** [comment]
Thanks @datapythonista

**Pull request merged**